cmd/process-agent/main.go

```diff
@@ -4,11 +4,49 @@ package main
 
 import (
 	"flag"
+	"github.com/DataDog/datadog-agent/cmd/agent/common"
+	"github.com/DataDog/datadog-agent/pkg/api/util"
```

Comment:
nit: we keep the core golang packages on top and separate out the external packages on the bottom with a newline between